aws_auth.tf

```diff
@@ -48,13 +48,13 @@ locals {
     {
       rolearn  = role["worker_role_arn"]
       username = "system:node:{{EC2PrivateDNSName}}"
-      groups = concat(
+      groups = tolist(concat(
```

Comment:
Maybe we should also add [`compact`](https://www.terraform.io/docs/configuration/functions/compact.html) function to remove the empty list before concat

Reply:
Isn't `contat([...], [])` enough to get rid of it?

Reply:
🤔 humm.. from what I'm reading in the example https://www.terraform.io/docs/configuration/functions/concat.html it sounds that concat doesn't remove empty elements from list, but compact does.

Maybe doc is outdated ?

Reply:
It won't remove empty string items, which is what the docs are showing. But concating an empty list won't add anything.

```
$ terraform console
> concat(["a"], [])
[
  "a",
]
>  
```

Reply:
Right